src/store.ts

```diff
@@ -123,16 +128,29 @@ state
     switch (manifest?.sdk()) {
       case 'rust':
         {
-          manifest
-            .overrideWorkspaceCommandConfig(
-              'rust-analyzer',
-              'server.path',
-              'rust-analyzer'
-            )
-            .then(
-              () => { }, // eslint-disable-line @typescript-eslint/no-empty-function
-              () => { } // eslint-disable-line @typescript-eslint/no-empty-function
-            )
+          if (workspace.getConfiguration(`${EXT_ID}`).get(ConfigValues.extensionsIntegration) == true) {
```

Comment:
This will need to be rebased and make use of the new functions provided by integration/rustAnalyzer.ts